doc-content/enterprise-only/cases/case-management-clean-case-log-proc.adoc

```diff
@@ -0,0 +1,61 @@
+[id='case-management-clean-case-log-proc']
+= Removing case logs from the database
+
+Use the `CaseLogCleanupCommand` to cleanup cases, such as canceled cases that lack value and that are using database space. The `CaseLogCleanupCommand` command consists of logic to clean up all or selected cases automatically. By default, the `CaseLogCleanupCommand` is executed once a day but can be configured to run on different intervals.
+
+There are several important configuration options that can be used with the `CaseLogCleanupCommand` command:
+
+.CaseLogCleanupCommand parameters table
+[cols="1,1,1", options="header"]
+|===
+| Name
+| Description
+| Is Exclusive
+
+| SkipProcessLog
+| Indicates if the clean-up of process instances, node instances and variables log cleanup should be omitted (default: false)
+| No, can be used with other parameters
+
+| SkipTaskLog
+| Indicates if the task audit and the task event log clean-up should be omitted (default: false)
+| No, can be used with other parameters
+
+| SkipExecutorLog
+| Indicates if the {PRODUCT} executor entries clean-up should be omitted (default: false)
+| No, can be used with other parameters
+
+| SingleRun
+| Indicates if the job routine should run only once (default: false)
+| No, can be used with other parameters
+
+| NextRun
+| Sets a date for the next run.
+
+For example, 12h is set for jobs to be executed every 12 hours.
+If the option is not given, the next job will run 24 hours after the completion of the current job
+| No, can be used with other parameters
+
+| OlderThan
+| Causes logs older than the given date to be removed. The date format is `YYYY-MM-DD`. Usually, this parameter is used for single run jobs
+| Yes, cannot be used when the OlderThanPeriod parameter is used
+
+| OlderThanPeriod
+| Causes logs older than the given timer expression should be removed. For example, set 30d to remove logs older than 30 day from current time
```

Comment:
Given or specified?

Reply:
Specified:

OlderThanPeriod
| Logs older than the specified timer expression are removed. For example, set 30d to remove logs older than 30 days.
| Yes, cannot be used when the `OlderThan` parameter is used